docs/DeveloperGuide.adoc

```diff
@@ -218,6 +218,42 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+// tag::removeTag[]
+=== removeTag command
+
+The removeTag command removes a tag with the specified name from the address book. All the appearances of the tag in
+the address book are removed. This command is implemented in the style of the `edit` command and `delete` command.
+
+=== Design Considerations
```

Comment:
Sorry for not noticing this earlier but! this should be **four** equal signs, not three...